setup/symfony_server.rst

```diff
@@ -50,6 +50,9 @@ run the Symfony server in the background:
 
     # start the server in the background
     $ symfony server:start -d
+    
+    # you can also use this shortcut
+    $ symfony serve -d
```

Comment:
As this is an alias we don't want to promote it. We did the same on the SymfonyCloud docs a few weeks ago. So I am against merging this too, sorry! 😢